src/test/java/com/iota/iri/profiling/RocksDbSizeProfiling.java

```diff
@@ -0,0 +1,142 @@
+                }
+                size += approvee.size();
+                approvee = approvee.next(localTangle);
+            }
+            // Key + # of entries in each value ( + delimiter)
+            sizes.add(counter * Hash.SIZE_IN_BYTES + size * (Hash.SIZE_IN_BYTES + 1));
+
+            ApproveeViewModel bundle = ApproveeViewModel.first(localTangle);
+            counter = 0;
+            size = 0;
+            while (bundle != null) {
+                if (++counter % 10000 == 0) {
+                    log.info("Scanned {} Bundles", counter);
+                }
+                size += bundle.size();
+                bundle = bundle.next(localTangle);
+            }
+            // Key + # of entries in each value ( + delimiter)
+            sizes.add(counter * Hash.SIZE_IN_BYTES + size * (Hash.SIZE_IN_BYTES + 1));
+
+            MilestoneViewModel milestone = MilestoneViewModel.first(localTangle);
+            counter = 0;
+            while (milestone != null) {
+                if (++counter % 10000 == 0) {
+                    log.info("Scanned {} Bundles", counter);
+                }
+                milestone = milestone.next(localTangle);
+            }
+            sizes.add(counter * (Integer.BYTES + Hash.SIZE_IN_BYTES));
+
+            MilestoneViewModel milestoneForStateDiff = MilestoneViewModel.first(localTangle);
+            counter = 0;
+            size = 0;
+            while (milestoneForStateDiff != null) {
+                if (++counter % 10000 == 0) {
+                    log.info("Scanned {} StateDiffs", counter);
+                }
+                StateDiffViewModel stateDiff = StateDiffViewModel.load(localTangle, milestoneForStateDiff.getHash());
+                size += stateDiff.getDiff().size();
+                milestoneForStateDiff = milestoneForStateDiff.next(localTangle);
+            }
+
+            sizes.add(counter * (Long.BYTES + Hash.SIZE_IN_BYTES) + size * (Long.BYTES + Hash.SIZE_IN_BYTES));
+
+            double sum = sizes.stream().reduce((a, b) -> a + b).get();
+            int i = 0;
+            log.info("----------------------------");
+            log.info(String.format("transactionBytes: %.2f", sizes.get(i++) / sum * 100));
+            log.info(String.format("addressBytes: %.2f", sizes.get(i++) / sum * 100));
+            log.info(String.format("tagBytes: %.2f", sizes.get(i++) / sum * 100));
+            log.info(String.format("approveeBytes: %.2f", sizes.get(i++) / sum * 100));
+            log.info(String.format("bundleBytes: %.2f", sizes.get(i++) / sum * 100));
+            log.info(String.format("milestoneBytes: %.2f", sizes.get(i++) / sum * 100));
+            log.info(String.format("stateDiffBytes: %.2f", sizes.get(i++) / sum * 100));
+            log.info(String.format("Total (uncompressed): %.2f GB", sum / 1073741824 /* GB */));
+        } finally {
+            log.info("----------------------------");
+            localTangle.shutdown();
```

Comment:
maybe add a startup again, then log the db size on disk as well (compressed) and shutdown again.
as this restart forces rocksdb to write to disk :D

Code could be cleaner by reusing a new method for each/most VMs by accepting name, first vm and size to increase